src/CpuAffinitySet.cc

```diff
@@ -38,7 +38,10 @@ CpuAffinitySet::apply()
     } else {
         cpu_set_t cpuSet;
         memcpy(&cpuSet, &theCpuSet, sizeof(cpuSet));
-        CPU_AND(&cpuSet, &cpuSet, &theOrigCpuSet);
+        // hack for clang on ubuntu-xenial, where CPU_AND returns a value
+        // which under very strict compiler checks fails the build
+        auto s = CPU_AND(&cpuSet, &cpuSet, &theOrigCpuSet);
+        (void) s;
```

Comment:
Can we do this in a single line?

```suggestion
        (void) CPU_AND(&cpuSet, &cpuSet, &theOrigCpuSet);
```

Reply:
No. That was the state before commit b2188e4d6bce80f2f93e3f99e11f3d054f5e8b60 and it wasn't working. That change broke xenial/clang instead